### PR TITLE
Update init script documentation

### PR DESCRIPTION
**Pull request by tomasonjo**
Adding multiple cypher call example and change the config option to be less ambiguous as it references to the Neo4j configuration file.

### CODE REVIEW

File: docs/asciidoc/modules/ROOT/pages/operational/init-script.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 Apoc optionally allows you to run a cypher command after database initialization is finished.
 This can e.g. be used to ensure indexes/constraints are created up front.
 
-To use this feature use a config option:
+To use this feature, add the following line to Neo4j configuration file:
```
Comment (sarmbruster) [inline docs/asciidoc/modules/ROOT/pages/operational/init-script.adoc:10]: it doesn't have to be necessarily in the config file. We can use either `apoc.conf`, env variables or system properties

Comment (tomasonjo) [reply docs/asciidoc/modules/ROOT/pages/operational/init-script.adoc:10]: words are hard

Comment (tomasonjo) [reply docs/asciidoc/modules/ROOT/pages/operational/init-script.adoc]: I have added the link to the configuration page

Comment (tomasonjo) [reply docs/asciidoc/modules/ROOT/pages/operational/init-script.adoc]: Obviously I don't know how to link properly, if you could help me with that would be great!

**Comment by tomasonjo** [comment]
I have just added the cypher.1 and cypher.2 options and removed the linking:

@sarmbruster

**Comment by conker84** [comment]
Thank you @tomasonjo !

**Pull request merged**